exercises/an_important_rule/problem.md

```diff
@@ -44,22 +44,6 @@ It is worth pointing out that both the synchronous **and** asynchronous
 code have the same problem.  If the rejection handler itself throws
 an error you are going to have a bad time.
 
```

Comment:
It may be a good time to mention the [promise events on `process`](https://nodejs.org/api/process.html#process_event_unhandledrejection) in this exercise, rather than the blurb about errors bubbling up to the global context, which I don't think is quite accurate.